lib/rules/comma-style.js

```diff
@@ -144,15 +145,21 @@ module.exports = {
                     : "between";
 
                 // lone comma
-                context.report({
-                    node: reportItem,
-                    loc: {
-                        line: commaToken.loc.end.line,
-                        column: commaToken.loc.start.column
-                    },
-                    messageId: "unexpectedLineBeforeAndAfterComma",
-                    fix: getFixerFunction(styleType, previousItemToken, commaToken, currentItemToken)
-                });
+                if (arrayLiteral) {
+
+                    // ignored element, move on.
```

Comment:
Place holder in case we may want to report other things later, based on new options.